mod-audit/src/main/resources/domain/mod-audit/features/undefined.feature

```diff
@@ -0,0 +1,73 @@
+Feature: mod audit data CRUD/errors
+
+  # CRUD
+
+  @Undefined
+  Scenario: Get a list of all audit records
+    * print 'undefined'
+
+  @Undefined
+  Scenario: Create audit record
+    * print 'undefined'
+
+  @Undefined
+  Scenario: Get audit record by id
+    * print 'undefined'
+
+  @Undefined
+  Scenario: Update audit record by id
+    * print 'undefined'
+
+  @Undefined
+  Scenario: Delete audit record by id
+    * print 'undefined'
+
+  @Undefined
+  Scenario: Get all circulation logs
+    * print 'undefined'
+
+  @Undefined
```

Comment:
Please remove tenants endpoint checking, as they provide some common logic.

Reply:
Removed, thank you.